Fix:

error: implicitly declaring library function 'exit' with type 'void (int) __attribute__((noreturn))' [-Werror,-Wimplicit-function-declaration]
error: implicitly declaring library function 'memset' with type 'void *(void *, int, unsigned long)' [-Werror,-Wimplicit-function-declaration]
error: implicitly declaring library function 'strcat' with type 'char *(char *, const char *)' [-Werror,-Wimplicit-function-declaration]
error: implicitly declaring library function 'strcmp' with type 'int (const char *, const char *)' [-Werror,-Wimplicit-function-declaration]
error: implicitly declaring library function 'strcpy' with type 'char *(char *, const char *)' [-Werror,-Wimplicit-function-declaration]
error: implicitly declaring library function 'strlen' with type 'unsigned long (const char *)' [-Werror,-Wimplicit-function-declaration]
error: implicitly declaring library function 'strncmp' with type 'int (const char *, const char *, unsigned long)' [-Werror,-Wimplicit-function-declaration]
error: implicitly declaring library function 'strncpy' with type 'char *(char *, const char *, unsigned long)' [-Werror,-Wimplicit-function-declaration]
--- configure.orig	2006-04-27 13:27:08.000000000 -0500
+++ configure	2021-01-23 02:17:32.000000000 -0600
@@ -3069,6 +3069,7 @@
 cat >>conftest.$ac_ext <<_ACEOF
 /* end confdefs.h.  */
 #include <ctype.h>
+#include <stdlib.h>
 #if ((' ' & 0x0FF) == 0x020)
 # define ISLOWER(c) ('a' <= (c) && (c) <= 'z')
 # define TOUPPER(c) (ISLOWER(c) ? 'A' + ((c) - 'a') : (c))
--- lib/rbmake/rbmake.h.orig	2006-04-23 17:49:47.000000000 -0500
+++ lib/rbmake/rbmake.h	2021-01-23 02:26:00.000000000 -0600
@@ -9,6 +9,7 @@
 #define _RBMAKE_H
 
 #include <stdio.h>
+#include <string.h>
 #include <rbmake/rbfile.h>
 
 #define RB_TEXTCONV_NONE	0
